plugins-ref/premium/powerpaste/introduction_to_powerpaste.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Introduction to PowerPaste
+title_nav: Introduction
+description: The PowerPaste plugin automatically cleans up content from Microsoft Word, Microsoft Excel, Google Docs, and HTML sources.
+keywords: enterprise powerpaste power microsoft word excel google docs
```

Comment:
@tylerkelly13 I edited these keywords a bit - removed option names and added gdocs. Did that elsewhere because copy+paste too. That okay? Should prob add the option names back to the options page I guess
- [ ] do that

Reply:
All good, thanks!